views/default/js/elgg/admin/upgrade.js

```diff
@@ -0,0 +1,225 @@
+
+define(function(require) {
+	var $ = require('jquery');
+	var elgg = require('elgg');
+
+	var upgrades = $('.elgg-upgrade');
+	var upgrade;
+	var className;
+	var progressbar;
+	var upgradeStartTime;
+	var timer;
+	var counter;
+	var percent;
+	var errorMessages = [];
+	var offset = 0;
+	var numSuccess;
+	var numError;
+	var numProcessed = 0;
+	var total;
+
+	/**
+	 *
+	 */
+	init = function() {
+		$('#upgrade-run').click(run);
```

Comment:
$('#upgrade-run').removeClass('hidden').click(run);

Show the button once js is loaded.